datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_LINEAGE_ID,
+        selector: `[id^='rc-tabs'][id$='Lineage']`,
+        title: 'Lineage Tab',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can view an entity&apos;s <strong>Lineage</strong> on this tab.
+                </p>
+                <p>
+                    Data <strong>Lineage</strong> allows you to visualize and understand both the upstream dependencies
+                    and downstream consumers of this entity.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_SCHEMA_ID,
+        selector: `[id^='rc-tabs'][id$='Schema']`,
+        title: 'Schema Tab',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can view a Dataset&apos;s <strong>Schema</strong> on this tab.
+                </p>
+                <p>
+                    You can also view or add <strong>Documentation</strong>, <strong>Tags</strong>, and{' '}
+                    <strong>Glossary Terms</strong> for specific columns.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_OWNERS_ID,
+        selector: `#${ENTITY_PROFILE_OWNERS_ID}`,
+        title: 'Owners',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can view and add <strong>Owners</strong> to this asset here.
+                </p>
+                <p>
+                    <strong>Owners</strong> are <strong>Users</strong> or <strong>Groups</strong> who are responsible
+                    for managing this asset.
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_TAGS_ID,
+        selector: `#${ENTITY_PROFILE_TAGS_ID}`,
+        title: 'Tags',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can view and add <strong>Tags</strong> to this asset here.
+                </p>
+                <p>
+                    <strong>Tags</strong> are labels for organizing your data. For example, you can add a Tag marking an
+                    asset as <strong>Deprecated</strong>.
+                </p>
+                <p>
+                    Learn more about <strong>Tags</strong>{' '}
+                    <a target="_blank" rel="noreferrer noopener" href="https://datahubproject.io/docs/tags">
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_GLOSSARY_TERMS_ID,
+        selector: `#${ENTITY_PROFILE_GLOSSARY_TERMS_ID}`,
+        title: 'Glossary Terms',
+        content: (
+            <Typography.Paragraph>
```

Comment:
This inline HTML is going to become very unmanagable. My rec would be to start a convention about how to better organize these. (not blocking)

Reply:
Added a Todo at the top to refactor this into multiple files